pydocx/openxml/wordprocessing/run.py

```diff
@@ -29,18 +17,19 @@ class Run(XmlModel):
     properties = XmlChild(type=RunProperties)
 
     children = XmlCollection(
-        EmbeddedObject,
-        TabChar,
-        Break,
-        NoBreakHyphen,
-        Text,
-        Drawing,
-        Picture,
-        DeletedText,
-        FootnoteReference,
-        FootnoteReferenceMark,
-        FieldChar,
-        FieldCode,
+        'wordprocessing.EmbeddedObject',
+        'wordprocessing.TabChar',
+        'wordprocessing.Break',
+        'wordprocessing.NoBreakHyphen',
+        'wordprocessing.Text',
+        'wordprocessing.Drawing',
+        'wordprocessing.Picture',
+        'wordprocessing.DeletedText',
+        'wordprocessing.FootnoteReference',
+        'wordprocessing.FootnoteReferenceMark',
+        'wordprocessing.FieldChar',
+        'wordprocessing.FieldCode',
+        'markup_compatibility.AlternateContent',
```

Comment:
Could you verify that is the case? We'll also need to ensure that any future additions have the requisite test cases as well.